javascript/2-functions/hw-3/scripts.js

```diff
@@ -1 +1,19 @@
 // Your JS code is here
+
+
+function calc(num, txt) {
+  if (num = 34 && txt != 'groups') {
+    return
+      console.log('option one is true')
+  } else if (num != 34 || txt = 'peoples') {
+    return
+      console.log('option two is true')
+  } else if (num = 534 || txt = 'students') {
+    return
+      console.log('option three is true')
+  }
+}
+calc(34, 'students')
+
```

Comment:
Please double check your function. You have several issues.
- check conditions. are you assigning or comparing ?
- also check where is your console log is located ? run this code and see if you will see any console log? if not why and fix it .